### PR TITLE
Tag release on-duty is error-issue bot issues

### PR DESCRIPTION
**Pull request by rcebulko**
Uses the new groups added in https://github.com/ampproject/amp-github-apps/issues/778 to @ mention the release on-duty within error report issues

### CODE REVIEW

**Comment by danielrozenberg** [comment]
This PR is closed, were you meaning to reopen it or just restore the branch while you're working on it?

**Comment by rcebulko** [comment]
> This PR is closed, were you meaning to reopen it or just restore the branch while you're working on it?

The former

File: error-issue/redacted.env [hunk 1]
```diff
@@ -2,3 +2,4 @@ GITHUB_REPO=ampproject/amphtml
 GITHUB_ACCESS_TOKEN=[REDACTED]
 PROJECT_ID=amp-error-reporting
 ERROR_ISSUE_ENDPOINT=https://us-central1-amp-error-issue-bot.cloudfunctions.net/error-issue
+RELEASE_ONDUTY=ampproject/release-on-duty
```
Comment (danielrozenberg) [inline error-issue/redacted.env:5]: nit: RELEASE_ONDUTY_TEAM

Comment (rcebulko) [reply error-issue/redacted.env:5]: Doesn't have to be a team, could be the actual release onduty username

Comment (rcebulko) [reply error-issue/redacted.env:5]: Anything that can be @ mentioned is valid here. I initially had team, but removed it for this reason

Comment (danielrozenberg) [reply error-issue/redacted.env:5]: ok

**Pull request merged**